### PR TITLE
MAUI-2856-Prepare-UG-document-for-the-auto-scrolling-delta-support-in-.NET-MAUI-cartesian-chart

### PR DESCRIPTION
**Pull request by NitheeshkumarThangaraj**
### Description ###
User Guide for the auto scrolling delta support in .NET Maui cartesian chart

### Solution Description
- I Created the UG Document in md Format named as AutoScrollingDelta.md.
- I added the Auto scrolling delta in the html List file.

### CODE REVIEW

**Comment by Saravanan-Madhesh** [comment]
@NitheeshkumarThangaraj rebase the pr.

File: MAUI/Cartesian-Charts/Axis/AutoScrollingDelta.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Auto scrolling delta in .NET MAUI Chart control | Syncfusion
+description: Learn here all about auto scrolling delta and its features in Syncfusion .NET MAUI Chart (SfCartesianChart) control and more.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# AutoScrollingDelta in .NET MAUI Chart
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/AutoScrollingDelta.md:10]: Auto scrolling in .NET MAUI Chart

File: MAUI/Cartesian-Charts/Axis/AutoScrollingDelta.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+[AutoScrollingDelta]() ensures that the specified range of data is always visible in the chart. It always shows the recently added data points at the end, and scrolling will be reset to the end of the range whenever a new point is added.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Axis/AutoScrollingDelta.md:12]: The auto-scrolling of [SfCartesianChart]() ensures that the specified range of data is always visible in the chart. It always shows the recently added data points at the end, and scrolling will be reset to the end of the range whenever a new point is added. The [AutoScrollingDelta ]() property of the chart axis can be used to set the number of data points to be always visible in the chart.

File: maui-toc.html [hunk 3]
```diff
@@ -177,6 +177,7 @@
                     <li><a href="/maui/cartesian-charts/axis/axisline">Axis Line</a></li>
                     <li><a href="/maui/cartesian-charts/axis/padding">Axis Padding</a></li>
                     <li><a href="/maui/cartesian-charts/axis/range-padding">Axis Range Padding</a></li>
+                    <li><a href="/maui/cartesian-charts/axis/autoscrollingdelta">Auto Scrolling Delta</a></li>
```
Comment (Saravanan-Madhesh) [inline maui-toc.html:180]: Auto Scrolling

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Axis/AutoScrollingDelta.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Auto scrolling delta in .NET MAUI Chart control | Syncfusion
```
[the remark this answers is not in the document]
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Axis/AutoScrollingDelta.md]: I changed the listed changes

File: MAUI/Cartesian-Charts/Axis/AutoScrollingDelta.md (hunk 2 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Axis/AutoScrollingDelta.md]: Replaced the above content

File: maui-toc.html (hunk 3 above)
Comment (NitheeshkumarThangaraj) [reply maui-toc.html]: Changed

**Review by Saravanan-Madhesh — APPROVED** [review]

**Pull request merged**